apps/events-helsinki/src/hooks/useRHHCConfig.tsx

```diff
@@ -70,57 +74,24 @@ export default function useRHHCConfig() {
           <VenueDetails {...(props as VenueDetailsProps)} />
         ),
       },
-      fallbackImageUrls: [
```

Comment:
what happened to `fallbackImageUrls` and `copy` settings?

Reply:
See the beginning of the config-object. They come from the `commonConfig` 
```
    return {
      ...rhhcDefaultConfig,
      ...commonConfig,
```